{
  if (cause instanceof WorkflowException || cause instanceof PrincipalException) {
    return true;
  }
  return false;
}
